.travis.yml

```diff
@@ -1,5 +1,21 @@
+os: linux
+dist: focal
+language: go
+cache:
+  directories:
+  - "$HOME/gopath/pkg/mod"
 jobs:
   include:
+  # - stage: tests
+  #   name: Run Tests
+  #   if: env(TEST) IS NOT present AND type = push AND tag IS NOT present
+  #   language: go
+  #   go:
+  #   - 1.17.x
+  #   git:
+  #     depth: false
+  #   script:
+  #   - make test || travis_terminate 1
```

Comment:
Comments?

Reply:
I tried implementing the tests for TravisCI but I ended up removing it because the tests don't work in the CI environment. Left it there so the person who picks up the next ticket to implement has a starting point